src/style.css

```diff
@@ -95,24 +90,32 @@ header {
   color: white;
 }
 
-.img-container {
-  display: flex;
-  flex-wrap: wrap;
+/* PopUp CSS */
+
+#commentPopup {
+  position: fixed;
+  height: 100vh;
+  width: 100vw;
+  display: none;
   justify-content: center;
-  width: 129px;
-  margin-left: 130px;
-  margin-right: 130px;
-  margin-top: 96px;
-  margin-bottom: -65px;
+  align-items: center;
+  background-color: rgba(0, 0, 0, 0.9);
 }
 
-#pokeimg {
-  width: 201px;
+.popup-container {
+  display: flex;
+  flex-direction: column;
+  background-color: green;
+  gap: 1em;
+  padding: 2em;
 }
 
-.poke-container {
-  display: flex;
-  flex-wrap: wrap;
-  justify-content: center;
-  margin: auto;
+#closePopup {
+  position: absolute;
+  top: 6em;
+  right: 26em;
+}
+
```

Comment:
- [ ] Correct the styles for closePopup to see the button inside the container.

![image](https://user-images.githubusercontent.com/58642949/171848107-5686b3e0-b6c8-4137-84c2-d390628a774e.png)